python/ql/test/experimental/dataflow/coverage/test.py

```diff
@@ -1,20 +1,23 @@
-# This should cover all the syntactical constructs that we hope to support
+# This should cover all the syntactical constructs that we hope to support.
+# Headings refer to https://docs.python.org/3/reference/expressions.html,
+# and are selected whenever they incur dataflow.
 # Intended sources should be the variable `SOURCE` and intended sinks should be
 # arguments to the function `SINK` (see python/ql/test/experimental/dataflow/testConfig.qll).
 #
 # Functions whose name ends with "_with_local_flow" will also be tested for local flow.
+#
+# All functions starting with "test_" should run and print a source (sources are defined in testConfig.qll).
 
-# These are included so that we can easily evaluate the test code
+# These are defined so that we can evaluate the test code.
 SOURCE = "source"
 def SINK(x):
     print(x)
 
 def test_tuple_with_local_flow():
     x = (3, SOURCE)
     y = x[1]
-    SINK(y)
+    SINK(y) # Flow missing
```

Comment:
How about testing the negative case as well, e.g. `SINK(x[0])`

Reply:
As you just pointed out, this comment is wrong - there is flow!

Reply:
Flow is coming to this one, by the shared magic of field flow, in a later PR.

Reply:
Ah, I see.

Reply:
Btw, with the current experiment, it will also flow to `x[0]`, so yes that negative test case should be there to document that fact.

Reply:
Done